inlong-manager/manager-dao/src/main/resources/mappers/InlongClusterEntityMapper.xml

```diff
@@ -198,4 +198,19 @@
         from inlong_cluster
         where id = #{id,jdbcType=INTEGER}
     </delete>
+
+    <select id="selectAllClusters"
+            resultType="org.apache.inlong.manager.common.pojo.sortstandalone.SortSourceClusterInfo">
+        select name,
+               type,
+               cluster_tags as cluster_tag,
+               ext_tag,
+               ext_params
+        from inlong_cluster
+        <where>
+            is_deleted = 0
+            AND
+            type in ('PULSAR','KAFKA','TUBE')
```

Comment:
These types should be passed in from the outside.

Reply:
fixed , thx